yarn-project/Dockerfile.prod

```diff
@@ -11,7 +11,7 @@ ARG COMMIT_TAG=""
 # TODO: Use release-please to update package.json directly, and remove this!
 RUN ./scripts/version_packages.sh
 # Productionify. See comment in yarn-project-base/Dockerfile.
-RUN yarn workspaces focus @aztec/cli @aztec/aztec-sandbox @aztec/aztec-faucet @aztec/aztec.js --production && \
+RUN yarn workspaces focus @aztec/aztec-faucet @aztec/aztec.js --production && \
```

Comment:
@spypsy do you know if this is safe to do?

Reply:
The cli Docker image builds on top of `Dockerfile.prod` so removing this would break the `aztec-cli`.